.github/workflows/deploy.yml

```diff
@@ -22,9 +21,6 @@ jobs:
       postgres_unit_testing: true
       db_name: fsd_assess_store_test
       deploy_to_dev: ${{github.event.inputs.deploy_to_dev == 'true'}}
-      users: 2
-      spawn-rate: 2
-      run-time: 11s
```

Comment:
Can you re-add users, run-time and spawn-rate please? @gio-karunakaran

Some repos needs to have different locust configs for the performance tests. This will override what we have set in the workflows repo for this repo only.